tests/helpers/with-page-auth-required.test.ts

```diff
@@ -170,15 +170,6 @@ describe('with-page-auth-required ssr', () => {
       delete process.env.NEXT_PUBLIC_AUTH0_LOGIN;
     });
 
-    test('is a no-op when invoked as a client-side protection from the server', async () => {
```

Comment:
We don't need this test anymore since the same `withPageAuthRequired` import isn't used for SSR and CSR since v3